site/hxapi/payment-gateway/index.md

```diff
@@ -0,0 +1,25 @@
+---
+
+---
+
+# Payment Gateway
+
+[API Docs](/hxapi/) > [Payment Gateway](index)
+
+## Integration guide
+
+Please read the integration guide before attempting to make any requests to these endpoints. Without the appropriate setup, you will not be able to access any of the Payment Gateway functionality.
+
+The correct steps to get setup can be found in our [guide to getting setup.](integration)
+
+## Payment
+
+| Request | Endpoint | Method |
+| ------- | -------- | ------ |
+| Make Payment | [https://api.holidayextras.co.uk/v1/pay](pay) | POST |
+
+## Refund
+
+| Request | Endpoint | Method |
+| ------- | -------- | ------ |
+| Make Refund | [https://api.holidayextras.co.uk/v1/refund](refund) | POST |
```

Comment:
Wouldn't the path for both of these endpoints be something like `/payment-gateway/pay`. We'll need to get INF to map `/payment-gateway` paths to our internal service for that tho. Also we won't use the versioning as part of the URI but by having it in the accept request header.

Reply:
yep - I did this just for now, so I could write the docs. We can change this when its been confirmed exactly what domain we will be hosting the payment gateway on 👍

Reply:
Hi Duncan, can you please get one of the INT pod to review this as well - ideally myself. We have structured payment in a specific way in the Partner API docs as the majority of partners do not pass through the details, and we do not want new partners to be using these methods.